src/test/java/com/google/cloud/logging/logback/LoggingAppenderTest.java

```diff
@@ -199,10 +199,8 @@ public void testDefaultWriteOptionsHasExpectedDefaults() {
     Timestamp timestamp = Timestamp.ofTimeSecondsAndNanos(100000, 0);
     LoggingEvent loggingEvent = createLoggingEvent(Level.ERROR, timestamp.getSeconds());
     loggingAppender.doAppend(loggingEvent);
-
     assertThat(logNameArg.getValue()).isEqualTo(defaultWriteOptions[0]);
-    // TODO(chingor): Fix this test to work on GCE and locally
-    // assertThat(resourceArg.getValue()).isEqualTo(defaultWriteOptions[1]);
+    assertThat(resourceArg.getValue()).isEqualTo(defaultWriteOptions[1]);
```

Comment:
how you confirm this would be work on GCE?

Reply:
Right now there is not integration test available in this product so i have checked only system test case in locally which working fine.